.github/workflows/coder.yaml

```diff
@@ -52,6 +52,8 @@ jobs:
       - uses: actions/checkout@v3
       - name: Run ShellCheck
         uses: ludeeus/action-shellcheck@1.1.0
+        env:
+          SHELLCHECK_OPTS: --external-sources
```

Comment:
This resolved some warnings around unspecified inputs. See https://www.mankier.com/1/shellcheck#-x and https://github.com/koalaman/shellcheck/issues/902